Selenium_cource/home_work.rb

```diff
@@ -18,294 +20,297 @@
 subject_feature = "feature_issue"
 subject_support = "support_issue"
 
+#Open Home Page
+        def open_home_page
+          @browser.get 'http://demo.redmine.org'
+        end
+
 #Register_method
-    def self.create_account(browser, login, password)
+    def create_account(login, password, name)
 
 
       #fill fields
-            browser.find_element(class: "register").click
-            browser.find_element(id: "user_login").send_key login
-            browser.find_element(id: "user_password").send_key password
-            browser.find_element(id: "user_password_confirmation").send_key password
-            browser.find_element(id: "user_firstname").send_key "name_"+login
-            browser.find_element(id: "user_lastname").send_key "last_name_"+login
-            browser.find_element(id: "user_mail").send_key login + "@usermail.com"
-            Selenium::WebDriver::Support::Select.new(browser.find_element(:id, "user_language")).
+            @browser.find_element(class: "register").click
```

Comment:
Where can I find valid Home Work?
here I can see instance variable @browser
in other files there is global variable $browser
And by the way you've said that you created new repository to make it clean from not needed files